files/content/more_aggressive_potions/potion_aggressive_append.lua

```diff
@@ -48,7 +48,7 @@ pa.potions =
 			if itemcomp == nil then return end
 			ComponentSetValue2(itemcomp[1], "item_name", "$100000_percent_acid")
 			ComponentSetValue2(itemcomp[1], "always_use_item_name_in_ui", true)
-			ComponentSetValue2(itemcomp[1], "ui_description", "$100000_percent_acid_desc")
+			ComponentSetValue2(itemcomp[1], "ui_description", GameTextGet("$100000_percent_acid_desc", "$bee_movie_script")) --low chance to just make it the entire bee movie script lmao
```

Comment:
`ComponentSetValue2` calls for the same field twice in a row?

The GameTextGet call always returns `"$bee_movie_script"` as far as I can tell

```lua
> return GameTextGet("$100000_percent_acid_desc", "$bee_movie_script")
"$bee_movie_script"
```

Reply:
fair, might be worth optimising

Reply:
wait no wdym, lemme ask on discord, github is annoying